src/OpenLoco/Vehicles/RenameVehicle.cpp

```diff
@@ -46,18 +46,20 @@ namespace OpenLoco::Vehicles
         Vehicles::VehicleHead* vehicleHead = ThingManager::get<Vehicles::VehicleHead>(vehicleHeadId); // 004B65BD-004B65C0
 
         static loco_global<char[512], 0x0112CC04> _stringFormatBuffer;
+        char renameStringBuffer[36] = "";
+        memcpy(renameStringBuffer, staticRenameBuffer, sizeof(renameStringBuffer));
 
         auto args = FormatArguments::common(vehicleHead->var_44);                     // 004B65C6-004B65CA
         StringManager::formatString(_stringFormatBuffer, vehicleHead->var_22, &args); // 004B65D0-004B65E6
```

Comment:
This doesn't have to be the string format buffer. Can just make it a local buffer for this function. Then we can give it a nice name like `existingVehicleName`